packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -645,6 +646,61 @@ describe('AppSec Index', () => {
       })
     })
   })
+
+  describe('Metrics', () => {
+    const appsecNamespace = telemetryMetrics.manager.namespace('appsec')
+    let config
+
+    beforeEach(() => {
+      sinon.restore()
+
+      config = new Config({
+        appsec: {
+          enabled: true
+        }
+      })
+    })
+
+    afterEach(() => {
+      appsec.disable()
+
+      appsecNamespace.reset()
```

Comment:
I think that we should do it also `beforeEach`, can other non related test increment the `waf.init` metric and break these tests?